tox.ini

```diff
@@ -34,6 +34,7 @@ commands =
     sphinx-build -a -b spelling docs/ docs/_build/spelling
     sphinx-build -a -b html docs/ docs/_build/html
     sphinx-build -a -b linkcheck docs/ docs/_build/linkcheck
+    trial --rterror flocker.restapi.docs
```

Comment:
Does this really belong here? This build environment is for generating the documentation. This new step is testing some of our documentation  infrastructure as far as I can tell. Maybe move it to `[testenv]`.